app/controllers/games_controller.rb

```diff
@@ -37,14 +40,25 @@ def update
   end
 
   def destroy
-    Game.destroy(params[:id])
-    redirect_to "/games"
+    if Game.find(params[:id])
+      @game = Game.find(params[:id])
+      if current_user.id == @game.user_id
+        @games = Game.destroy(params[:id])
+        flash[:notice] = 'Game deleted.'
+        redirect_to action: "index"
+      else
+        flash[:notice] = "you can not delete other user's games"
+        render :edit
```

Comment:
Can a user edit another user's game? You're displaying the edit for for a game that the user doesn't own in this case.